Automattic/newspack-plugin
# Logo Update

### All Submissions:






### Changes proposed in this Pull Request:

Updates the logo to the revised version. 

<img width="1493" alt="Screen Shot 2019-08-21 at 6 32 17 PM" src="https://user-images.githubusercontent.com/1477002/63473057-2a6a3580-c442-11e9-8af3-3597208d87f0.png">

<img width="1371" alt="Screen Shot 2019-08-21 at 7 12 42 PM" src="https://user-images.githubusercontent.com/1477002/63474595-b6329080-c447-11e9-803d-29a4500f1359.png">

### How to test the changes in this Pull Request:

1. Check out the branch, run `npm run build:webpack`
2. Observe the revised logo in the admin sidebar.
3. Navigate to Setup Wizard, and observe the revised logo at the top.
4. Navigate to Dashboard, observe revised logo.
5. Navigate to Advertising Wizard, observe revised logo in compact form (logomark only).
6. Navigate to any Wizard that uses Handoff (e.g. Advertising->Ad Sense) and click the Handoff link. Observe the revised logo in the handoff banner.

### Other information:
